console/src/main/webapp/manager/app/components/user/user.es6

```diff
@@ -66,14 +66,32 @@ class UserController {
     return (this.delegation.orgs.length !== 0) && (this.delegation.roles.length !== 0)
   }
 
+  // search each choosen span elements and set title manually
+  // to display roles description on hover
+  setTitle () {
+    if (this.systemRoles) {
+      let selectEls = $('li.search-choice')
+      $(selectEls).each(i => {
+        let text = selectEls[i].firstChild.innerHTML
+        selectEls[i].firstChild.setAttribute('title', this.systemRoles[text])
+      })
+    }
+  }
+
   bindRoles () {
     const TMP_ROLE = 'TEMPORARY'
 
     // Load role infos for every tab (for confirmation)
     let Role = this.$injector.get('Role')
     this.roles = Role.query()
-    Role.query(roles => { this.allroles = roles.map(r => r.cn) })
-
+    Role.query(roles => {
+      this.allroles = roles.map(r => r.cn)
+      // get roles informations to get description from template
+      this.systemRoles = {}
+      roles.map(r => {
+        this.systemRoles[r.cn] = r.description
```

Comment:
either rename it to `roleDescriptions` or store the whole role rather than description.